### PR TITLE
arch/risc-v/src/mpfs: Set USB DMA upper addr offset

### PR DESCRIPTION
**Pull request by jpaali**
Use a configuration register to set the upper address lines [37:32] for USB DMA engine.

### CODE REVIEW

File: arch/risc-v/src/mpfs/mpfs_usb.c [hunk 1]
```diff
@@ -129,6 +129,10 @@
 #  define MSB 1
 #endif
 
+#ifndef MPFS_USB_DMA_ADDR_UPPER_OFFSET
+#  define MPFS_USB_DMA_ADDR_UPPER_OFFSET 0x14u
```
Comment (sthirvela) [inline arch/risc-v/src/mpfs/mpfs_usb.c:133]: Extra space

File: boards/risc-v/mpfs/icicle/include/board.h [hunk 2]
```diff
@@ -45,6 +45,8 @@
 #  define MPFS_SD_CLOCK_4BIT MPFS_MMC_CLOCK_25MHZ
```
Comment (sthirvela) [inline boards/risc-v/mpfs/icicle/include/board.h:45]: Extra space

File: arch/risc-v/src/mpfs/mpfs_usb.c (hunk 1 above)
Comment (pussuw) [reply arch/risc-v/src/mpfs/mpfs_usb.c:133]: These are according to the nuttx coding standard, indentation here is correct

**Review by pussuw — APPROVED** [review]
LGTM

File: arch/risc-v/src/mpfs/mpfs_usb.c (hunk 1 above)
Comment (jpaali) [reply arch/risc-v/src/mpfs/mpfs_usb.c:133]: Done similarly it has been done earlier in that file.

**Pull request merged**